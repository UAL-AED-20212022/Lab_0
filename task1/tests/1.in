S 10 5
S 50 10
S 2
SU 5 10
SU 20 4
SU 15 350
M 2
M 3 6
M 4 6
M 4
D 12 2
D 100 2
F 2
F 5
F 8