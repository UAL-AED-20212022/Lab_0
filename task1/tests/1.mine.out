O resultado é 15.
O resultado é 60.
Não é possível realizar a operação.
Não é possível realizar a operação.
O resultado é 16.
Não é possível realizar a operação.
Não é possível realizar a operação.
O resultado é 18.
O resultado é 24.
Não é possível realizar a operação.
O resultado é 6.0.
O resultado é 50.0.
O resultado é 2.
O resultado é 120.
O resultado é 40320.
